src/test/scala/kiambogo/scrava/IntegrationTest.scala

```diff
@@ -90,12 +90,12 @@ class IntegrationTest extends FlatSpec with Matchers {
 
   it should "retrieve friends activities" in {
     val activities = testClient.listFriendsActivities()
-    activities.size should equal(30)
+    activities.size should equal(27)
```

Comment:
I don't know what happened here but this is independent of my code,  i.e. this was failing even in commit fd1de39 of 2017-08-02.

I'm assuming this is because the "retrieve friends activities" only retrieves recent activities, as they say in the [API documentation](https://strava.github.io/api/v3/activities/#get-feed).

Reply:
Yes, no problem. I've had to change it a few times already 👍